src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_NullChecking.cs

```diff
@@ -43,6 +43,17 @@ private BoundBlock RewriteNullChecking(BoundBlock block)
 
         }
 
+        internal static bool GetParameterIsNullChecked(ParameterSymbol x)
```

Comment:
>GetParameterIsNullChecked [](start = 29, length = 25)

Is this method needed? Isn't `IsNullChecked` defined on `ParameterSymbol` directly?